spec/expected/help.menu.txt

```diff
@@ -1,5 +1,5 @@
 Precompile handlebar templates.
-Usage: handlebars [template|directory]...
+Usage: handlebars.js [template|directory]...
```

Comment:
:warning: This shouldn't change.

Reply:
This works as intended. From the `yargs` docs:

*.usage(, [desc], [builder], [handler])*
```
Set a usage message to show which commands to use. Inside message, the string $0 will get interpolated to the current script name or node command for the present script similar to how $0 works in bash or perl.
```

Since npm aliases the bin-script with a without a `.js`-extension, the output won't show `.js` if called via npm/npx.